modules/caddyhttp/reverseproxy/fastcgi/caddyfile.go

```diff
@@ -179,7 +179,7 @@ func parsePHPFastCGI(h httpcaddyfile.Helper) ([]httpcaddyfile.ConfigValue, error
 	indexFile := "index.php"
 
 	// set up for explicitly overriding try_files
-	tryFiles := []string{}
+	var tryFiles []string
```

Comment:
About to head out, but initializing as nil might have different implications... can't remember if there was a reason we explicitly initialized as an empty, non-nil slice.